### PR TITLE
skivi: Use qtpy to handle different Qt versions

### PR DESCRIPTION
**Pull request by hmaarrfk**
qtpy is MIT Licensed.

## Description
skivi was completely broken.

Now I only get the following warnings:
```
QWindowsWindow::setGeometry: Unable to set geometry 150x46+1600+800 on QWidgetWindow/'QPushButtonClassWindow'. Resulting geometry:  232x46+1600+800 (frame: 13, 58, 13, 13, custom margin: 0, 0, 0, 0, minimum size: 0x0, maximum size: 16777215x16777215).
QWindowsWindow::setGeometry: Unable to set geometry 150x46+1600+800 on QWidgetWindow/'QPushButtonClassWindow'. Resulting geometry:  232x46+1600+800 (frame: 13, 58, 13, 13, custom margin: 0, 0, 0, 0, minimum size: 0x0, maximum size: 16777215x16777215).
```
And I can change the colors (used `floordiv` instead of `truediv`). I haven't tried saving.

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
I searched for pyqt4 in the repo and couldn’t find other references.

Maybe I missed them

**Comment by hmaarrfk** [comment]
@soupault, viewer uses lots of matplotlib. I dont know too much about it. 

I do suggest offloading the qt compatibility to qtpy (and not doing it ourselves).

I think the way to go about it would be to rip off `skimage.viewer.qt.py` and seeing where the errors pop up.

Also this doesn't look right.

image

**Comment by hmaarrfk** [comment]
Some final remarks for the viewer:

unfortunately, it uses `FigureManagerQT` which isn't part of qtpy. so i dont know what to do about that.

File: doc/release/release_dev.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ New Features
 Improvements
 ------------
 
-
+- qtpy (MIT) now an optional dependency for skivi to run properly.
```
Comment (soupault) [inline doc/release/release_dev.rst]: I cannot push to your branch, but I'd like to reword this to something like:
```
- ``skivi`` is now using ``qtpy`` for Qt4/5 compatibility (a new optional dependency). 
```
before merging.

**Comment by soupault** [comment]
Great work @hmaarrfk ! Thanks!

**Comment by soupault** [comment]
We should also update `INSTALL.rst` to mention `qtpy`.

**Comment by hmaarrfk** [comment]
@scikit-image/core  backport?

I understand you wouldn't want to add a dependency, but seeing as 0.14 is LTS, maybe this makes sense?

**Comment by stefanv** [comment]
@meeseeksdev backport to v0.14.x
